src/createSession.js

```diff
@@ -1,3 +1,35 @@
+import axios from 'axios';
+
+const libraryPackage = require('../package.json');
+
+const logAnalyticsEvent = (apiKey, sessionId, action, connectionId) => {
```

Comment:
IMHO, the logAnalytics should be moved out of createSession file, where we have a constructor to init that and a couple of methods to log client event and to log client error . It could be reused in the future.

Reply:
I agree with this approach, would make it easier to make changes in the future as well

Reply:
Done, I've made it into a helper function!